project-deployer/python-runnables/dataiku_project_or_bundle_migration/runnable.py

```diff
@@ -38,7 +38,13 @@ def run(self, progress_callback):
             raise Exception("API key is required")
 
         activate_scenarios = self.config.get("activate_scenarios")
-        
+
+        ignore_proxy_conf = self.config.get("ignore_proxy_env_settings")
+
+        if ignore_proxy_conf:
+            os.environ["http_proxy"] = ""
```

Comment:
so actually probbly safer to 
`os.environ.pop('http_proxy', None)`

Reply:
That's fixed 😜